services/repository/src/main/java/org/sagebionetworks/repo/web/controller/EntityController.java

```diff
@@ -1424,7 +1423,13 @@ public void filePreviewRedirectURLForVersion(
 
 	/**
 	 * Gets all FileEntities whose file's MD5 is the same as the specified MD5
-	 * string.
+	 * string. NOTE: This will return a subset of at most 200 entity headers that is the
+	 * intersection of the set of the most recently created entity headers with the specified MD5 string and
+	 * the set of entities which the user has access to. To look at MD5 values of entities
+	 * in a specific project, create a file view.
```

Comment:
Not necessarily for a project only, I would just mention that for more flexibility a file view that includes the MD5 can be used.